### PR TITLE
implemented producer and consumer with avro format

### PR DESCRIPTION
**Pull request by SadineniAbhi**
this pr might be little sketchy since it already got half merged, check the avro folder all the avro related Task Files are in it.

### CODE REVIEW

File: avro/src/avro_consumer.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+            print(msg.error())
+        else:
+            msg = msg.value()
+            print(serializer.deserialize(msg))
+
+
+if __name__ == "__main__":
+    conf = get_config('config.ini')
+    con_obj = ConsumerFactory(conf['consumer']['group_id']).get_consumer()
+    schema = get_schema('user.avsc')
```
Comment (sadinenianil) [inline avro/src/avro_consumer.py:24]: What is get_schema doing ? read schema from file and return schema object ?

**Review by sadinenianil — APPROVED** [review]

File: avro/src/avro_consumer.py (hunk 1 above)
Comment (SadineniAbhi) [reply avro/src/avro_consumer.py:24]: found mistake in serializer class it is not serializing into avro it is serializing into byte stream will fix it now.
get schema is reading schema from the file

Comment (SadineniAbhi) [reply avro/src/avro_consumer.py:24]: it is correct babai, don't know anything about but the methods just copied them ,
the method just require the schema in json form. Merging now

Comment (SadineniAbhi) [reply avro/src/avro_consumer.py:24]: yes reads schema and returns dict object

**Pull request merged**